fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -1255,19 +1255,20 @@ private static void checkSearchParameterCombinations(Class<?> resourceType, List
                     parameterValue.setValueCode(unescapeSearchParm(parts[1]));
                 } else {
                     // Optimization for search parameters that always reference the same system, added under #1929
-                    try {
-                        String implicitSystem = searchParameter.getExtension().stream()
-                                .filter(e -> SearchConstants.IMPLICIT_SYSTEM_EXT_URL.equals(e.getUrl()) && e.getValue() != null)
-                                .findFirst()
-                                .map(e -> e.getValue().as(Uri.class).getValue())
-                                .orElse(null);
-                        if (implicitSystem != null) {
-                            parameterValue.setValueSystem(implicitSystem);
+                    if (!Modifier.MISSING.equals(modifier)) {
```

Comment:
ah, very interesting.  i definitely hadn't considered what might happen in here if there is a `:missing` modifier